### PR TITLE
Beta

### PR DESCRIPTION
**Pull request by monicachew**
Hmm, looks like the changeset links broke again :P @mozkeeler this is already pushed

### CODE REVIEW

**Comment by monicachew** [comment]
oops! fat finger accidentally merged :(

File: dashboard.js [hunk 1]
```diff
@@ -90,6 +94,31 @@ function sortByDate(p1, p2)
   return p1[0] - p2[0];
 }
 
+// Filter duplicate dates to account for screwed up telemetry data
+function filterDuplicateDates(series)
+{
+  // Work on a copy so we don't cause side-effects without realizing.
+  var s = series;
```
Comment (mozkeeler) [inline dashboard.js:101]: series.slice() would be a copy, right? (how do I edit in place again?)

**Comment by mozkeeler** [comment]
Other than that, LGTM :)

File: dashboard.js (hunk 1 above)
Comment (monicachew) [inline dashboard.js:101]: I think you can't for already merged PRs, but you can use the "Edit" button on the top right.

**Pull request merged**